ampd-handlers/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM rust:1.81-bookworm as builder
+
+WORKDIR /ampd-handlers
+COPY . .
+RUN cargo build --release --bin sample_handler
```

```suggestion
RUN cargo build --release --bin ampd_handler
```
so that we don't forget to rename it properly later.